gravityflow/gravityflowincomingwebhook
# Fix Acceptance Tests failing with Gravity Forms 2.5

## Description
Fix the failing Acceptance Tests, to conform to the new release of Gravity Forms (2.5).

## Testing instructions
1. Tests on `master` are failing with Gravity Forms 2.5
2. Switch to this branch and confirm all tests pass fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
